shape.c

```diff
@@ -157,12 +170,88 @@ rb_shape_frozen_shape_p(rb_shape_t* shape)
     return SHAPE_FROZEN == (enum shape_type)shape->type;
 }
 
-void
-rb_shape_transition_shape_remove_ivar(VALUE obj, ID id, rb_shape_t *shape)
+static void
+move_iv(VALUE obj, ID id, attr_index_t from, attr_index_t to)
+{
+    switch(BUILTIN_TYPE(obj)) {
+        case T_CLASS:
```

Comment:
nitpick: CRuby devs use 2-space indentation under `switch` and 2-space indentation under `case`, using 4 spaces between `switch` and `break` in total.

Reply:
Thanks. For some reason Vim won't indent it this way (I'm not sure why).  I'll fix it.

Reply:
Actually my (Neo)vim doesn't either. I just manually adjust them. It'd be nice to automate it 🤔

Reply:
I'll try to figure out a way.  I _want_ to indent it correctly, but I'm also lazy. 🤣